packages/mdc-drawer/README.md

```diff
@@ -2,23 +2,20 @@
 title: "Drawers"
 layout: detail
 section: components
-excerpt: "Permanent, persistent, and temporary drawers."
+excerpt: "Permanent drawers."
 iconId: side_navigation
 path: /catalog/drawers/
 -->
 
 # Drawers
 
-The MDC Drawer component is a spec-aligned drawer component adhering to the
-[Material Design navigation drawer pattern](https://material.io/go/design-navigation-drawer).
-It implements permanent, persistent, and temporary drawers. Permanent drawers are CSS-only and require no JavaScript, whereas persistent and temporary drawers require JavaScript to function, in order to respond to
-user interaction.
+The MDC Navigation Drawer is used to organize access to destinations and other functionality on an app.
 
 ## Design & API Documentation
 
 <ul class="icon-list">
   <li class="icon-list-item icon-list-item--spec">
-    <a href="https://material.io/go/design-navigation-drawer">Material Design guidelines: Navigation drawer</a>
+    <a href="https://material.io/design/components/navigation-drawer.html">Material Design guidelines: Navigation drawer</a>
```

Comment:
I think we'd prefer to keep the go link here, so the site team has flexibility to change things without these links breaking.